packages/worker/src/api/controllers/admin/auth.js

```diff
@@ -10,7 +10,7 @@ const { checkResetPasswordCode } = require("../../../utilities/redis")
 
 const GLOBAL_DB = authPkg.StaticDatabases.GLOBAL.name
 
-function authInternal(ctx, user, err = null) {
+async function authInternal(ctx, user, err = null) {
```

Comment:
How come we made this one async?

Reply:
Good catch @Rory-Powell - I had planned to do some async work in this function but ended up moving it further up the chain, forgot to put this back to sync, will fix on my next branch!

Reply:
ah okay that makes sense, sounds good